### ISSUE TITLE
How to crop to square from not square image?

### ISSUE THREAD

**Issue by sekitaka-RBS** [TakuSemba/CropMe#3]
Thank you for great library 🥇 

I'am trying to crop a image to square from not square image.
But cropme_result_width(height)  attribute only accept percent.
How can I specify cropme_result_width (height) using dp or px?

**Comment by TakuSemba** [comment]
can i see a screenshot when you specified `cropme_result_width ` with not squared image ???

**Comment by sekitaka-RBS** [comment]
Thanks for your reply.
Screenshot here.
![screenshot_20180222-090121](https://user-images.githubusercontent.com/21375055/36512801-65824c8e-17af-11e8-96ab-57ef4511d479.png)

I'm using this layout xml.
```
<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="match_parent">

    <com.takusemba.cropme.CropView
        android:id="@+id/cropView"
        android:layout_width="0dp"
        android:layout_height="0dp"
        app:cropme_max_scale="3"
        app:cropme_result_height="50%"
        app:cropme_result_width="50%"
        app:layout_constraintBottom_toBottomOf="parent"
        app:layout_constraintDimensionRatio="1:1"
        app:layout_constraintEnd_toEndOf="parent"
        app:layout_constraintStart_toStartOf="parent"
        app:layout_constraintTop_toTopOf="parent" />

</android.support.constraint.ConstraintLayout>
```

**Comment by TakuSemba** [comment]
oh, I understood. 

it seems to be a bug. 
I will fix it ASAP.

**Comment by TakuSemba** [comment]
It fixed at version 1.0.6!!!
